lib/matplotlib/colorbar.py

```diff
@@ -216,6 +219,47 @@ def _set_ticks_on_axis_warn(*args, **kw):
     warnings.warn("Use the colorbar set_ticks() method instead.")
 
 
+class ColorbarAutoLocator(ticker.MaxNLocator):
+    """ AutoLocator for Colorbar
+
+        This locator is just a `.MaxNLocator` except the min and max are
+        clipped by the norm's min and max (i.e. vmin/vmax from the
+        image/pcolor/contour object).  This is necessary so ticks don't
+        extrude into the "extend regions".
+    """
+
+    def __init__(self, colorbar, *args, **kwargs):
```

Comment:
Why are there args and kwargs here if they're not used? Can this also get a docstring (just for the colorbar parameter is probably fine)